models/orderDao.js

```diff
@@ -0,0 +1,157 @@
+const { dataSource } = require('./dataSource');
+
+const getOrderStatusId = async (orderStatus) => {
+  try {
+    const [orderStatusId] = await dataSource.query(
+      `
+          SELECT id
+              FROM order_status
+              WHERE status = ?
+          `,
+      [orderStatus]
+    );
+    return orderStatusId;
+  } catch (error) {
+    error = new Error('DATABASE_CONNECTION_ERROR');
+    error.statusCode = 400;
+    throw error;
+  }
+};
```

Comment:
status는 db에서 가져오지 말고 다음과 같이 enum화 하면 좋을 것 같습니다.
```suggestion
const orderStatus = {
  "pending" : 1,
  "complete" : 2
}

Object.freeze(orderStatus)
```